### DISCUSSION TITLE
Homeassistant user

### DISCUSSION THREAD

**Discussion by nosfaratu** [bimmerconnected/bimmer_connected#608] [Q&A] [answered]
Hello guys!
I have a problem, I don't know how to install this on HA...
If anyone could help me with "how to install an idiot" I would be grateful.
I already have the BMW connected drive installed with some entities available.
But I would like to have the rest of the information that appears on the website.
Thanks!

Edit---

I run my homeassistant on an unraid VM.

**Comment by sincze** [comment]
> Hello guys!
> I have a problem, I don't know how to install this on HA...
> If anyone could help me with "how to install an idiot" I would be grateful.
> I already have the BMW connected drive installed with some entities available.
> But I would like to have the rest of the information that appears on the website.
> Thanks!
> 
> Edit---
> 
> I run my homeassistant on an unraid VM.

If you mean tyre pressure and per door / window the lock status i am interested to.  Seems I only have a limited amount of devices in HA that is avaliable in the full JSON if started from CLI

**Comment by rikroe** [comment]
If you want more information sensors, please check in the [feature requests](https://github.com/bimmerconnected/bimmer_connected/discussions/categories/feature-request) if there is already an topic (and be quite specific please).
The more support an idea gets, the better.

Otherwise there is not much more to do than getting the integration running. We do not have too many entities/sensors hidden by default right now.

**Reply by nosfaratu** [reply]
The problem is I don't know how to install :/
 I use HA in vm in unraid on a mini pc.

Is there any tutorial on how to install for noobs?

**Reply by rikroe** [reply, marked as answer]
If you have HA running, nothing much to do. Just follow the HA documentation at
https://www.home-assistant.io/integrations/bmw_connected_drive/#configuration 

You might want to wait until HA 2024.5.0 due to fixes for API changes. 

If you want to have more data, please open feature requests in discussions on what data you're interested in.
